Tech-Lab-On-Campus/Resources/Git-Commands.md

```diff
@@ -0,0 +1,152 @@
+- [**Open Source**](https://opensource.com/resources/what-open-source) - The term open source refers to something people can modify and share because its design is publicly accessible.
+
+There are other distributed version control systems that are widely used. For example, Facebook uses Mercurial. Instead of typing `git` before commands, you would use `hg` for this version control system. The three most popular [DVCS (Distributed Version Control Systems)](https://www.atlassian.com/blog/software-teams/version-control-centralized-dvcs#:~:text=Distributed%20Version%20Control,-In%20the%20past&text=The%20three%20most%20popular%20of,on%20their%20own%20hard%20drive.) are Mercurial, Git, and Bazaar. Here, we will solely focus on Git.
+
+## What is GitHub
+
+GitHub is a cloud-based platform where you can store, share, and work together with others to write code.
+
+Storing your code in a "repository" on GitHub allows you to:
+
+- Showcase or share your work.
+- Track and manage changes to your code over time.
+- Let others review your code, and make suggestions to improve it.
+- Collaborate on a shared project, without worrying that your changes will impact the work of your collaborators before you're ready to integrate them.
+- Collaborative working, one of GitHub’s fundamental features, is made possible by the open-source software, Git, upon which GitHub is built.
+
+There are other code management platforms that rely on git such as GitLab. But GitHub is the most popular, with over 56 million users worldwide.
+
+## Creating your own Organizations and Repositories
+
+You can create a repository for each project you start on your personal GitHub account, [but many suggest creating a separate organization](https://medium.com/@ajlehechka/why-you-should-create-a-github-organization-for-your-side-project-d7c941dbb45b) for projects you plan to collaborate on or make open-source.
+
+#### [QuickStart tutorial for Repositories](https://docs.github.com/en/repositories/creating-and-managing-repositories/quickstart-for-repositories)
+
+#### [README Files](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes)
+
+- You can add a README file to your repository to tell other people why your project is useful, what they can do with your project, and how they can use it.
+- If a filed titled `README.md` file in your repository's hidden .github, root, or docs directory, GitHub will recognize and automatically surface your README to repository visitors.
+- If a repository contains more than one README file, then the file shown is chosen from locations in the following order: the .github directory, then the repository's root directory, and finally the docs directory.
+- [Markdown Syntax](https://www.markdownguide.org/basic-syntax/)
```

Comment:
Similar. It's under [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/) so I believe it's fine but it's owned by an individual so not sure the policy. Let's avoid for now
```suggestion
```